### ISSUE TITLE
Javascript error: IEDriver on Protect mode OFF

### ISSUE THREAD

**Issue by RaviAutomation** [dzharii/swd-recorder#44]
HI,

Due to security reasons and no admin rights, i tried to handle IE protect mode through script but which was failed to run using the tool.

After i handled IE protected mode through the code , tool giving java script error when i try to perform Ctrl+Right click on any element.

The below changes i did to run on IE,
**_var options = new InternetExplorerOptions();
 options.IntroduceInstabilityByIgnoringProtectedModeSettings = true;
options.InitialBrowserUrl = "https://google.com";
driver= new InternetExplorerDriver(options);_**

after this change, i was able to invoke internet explorer but not able to locate any element,and saw the above explained error message.

Note: Working fine on firefox.
![iejscripterror](https://cloud.githubusercontent.com/assets/1586248/12879485/d80c8ba8-ce59-11e5-8321-b4c27584992d.png)

Thanks In Advance.

**Comment by dzharii** [comment]
Hi RaviAutomation, 
This is why the setting `IntroduceInstabilityByIgnoringProtectedModeSettings` has 
this long name ;)

Could you please take a look (and send a screenshot if possible) to the F12 Internet Explorer Console? 
There should be an error after you started WebElementExplorer or after you performed Ctrl+RightClick. 

If that would be a SECURITY error, I am afraid nothing can be done, but let's see. 

@dzhariy

**Comment by RaviAutomation** [comment]
Please find the attachments.

![capture](https://cloud.githubusercontent.com/assets/1586248/12881668/891479da-ce71-11e5-99a9-5ca6cea0e27d.PNG)
[IEscript.txt](https://github.com/dzharii/swd-recorder/files/121326/IEscript.txt)

**Comment by dzharii** [comment]
Hi @RaviAutomation, 

I think I have found the cause of the issue. 

As a hotfix, could you please REPLACE 
the file:
`SwdPageRecorder_Latest\JavaScript\ElementSearch.js`
with the file, provided in the following gist:
https://gist.github.com/dzharii/137fef8e3f330ca8801b

the cause was a javascript call to the constant that does not exists in IE8. 
Your screenshot helped to reproduce the issue, please check if web element explorer works on your side after you have applied the hot-fix

**Comment by RaviAutomation** [comment]
Yes, its resolved and able to see element explorer but created new issue.
after created element explorer, when i clicked on the 'REFRESH' button getting java script error.please find the screenshot.
![refresherror](https://cloud.githubusercontent.com/assets/1586248/12910213/8c88ae70-cf2e-11e5-8080-5cf3beadb009.PNG)

**Comment by dzharii** [comment]
Sorry, @RaviAutomation I am unable to reproduce this issue. 

Do you use IE8 or had it happened on other browser? I'll try to experiment with the javascript executor to provide more reasonable error messages. For now I am not sure what had happened at all O_O

**Comment by RaviAutomation** [comment]
Its happening only for IE8,but working fine on Firefox.me too trying to get to know...am thinking its bcz of protect mode is hadling through script,instead of browser settings level.
